<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QOP64WindowClass</class>
 <widget class="QMainWindow" name="QOP64WindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>205</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>382</width>
    <height>205</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>382</width>
    <height>205</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>op64</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>382</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_ROM"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionGraphics_Settings"/>
    <addaction name="actionAudio_Settings"/>
    <addaction name="actionRSP_Settings"/>
    <addaction name="actionInput_Settings"/>
    <addaction name="separator"/>
    <addaction name="actionEmulator_Settings"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuAdvanced">
    <property name="title">
     <string>Advanced</string>
    </property>
    <addaction name="actionShow_Log"/>
   </widget>
   <widget class="QMenu" name="menuEmulation">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Emulation</string>
    </property>
    <widget class="QMenu" name="menuReset">
     <property name="title">
      <string>Reset</string>
     </property>
     <addaction name="actionSoft_Reset"/>
     <addaction name="actionHard_Reset"/>
    </widget>
    <addaction name="actionLimit_FPS"/>
    <addaction name="menuReset"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEmulation"/>
   <addaction name="menuOptions"/>
   <addaction name="menuAdvanced"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionOpen_ROM">
   <property name="text">
    <string>Open ROM</string>
   </property>
  </action>
  <action name="actionRSP_Settings">
   <property name="text">
    <string>RSP Settings</string>
   </property>
  </action>
  <action name="actionGraphics_Settings">
   <property name="text">
    <string>Graphics Settings</string>
   </property>
  </action>
  <action name="actionAudio_Settings">
   <property name="text">
    <string>Audio Settings</string>
   </property>
  </action>
  <action name="actionInput_Settings">
   <property name="text">
    <string>Input Settings</string>
   </property>
  </action>
  <action name="actionEmulator_Settings">
   <property name="text">
    <string>Emulator Settings</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionShow_Log">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Show Log</string>
   </property>
  </action>
  <action name="actionLimit_FPS">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Limit FPS</string>
   </property>
  </action>
  <action name="actionHard_Reset">
   <property name="text">
    <string>Hard Reset</string>
   </property>
  </action>
  <action name="actionSoft_Reset">
   <property name="text">
    <string>Soft Reset</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qop64window.qrc"/>
 </resources>
 <connections/>
</ui>
